DcCore/DcCore/DC/events.swift

```diff
@@ -56,118 +56,105 @@ public class DcEventHandler {
             }
             logger.info("📡[\(accountId)] chat modified: \(data1)")
-            DispatchQueue.main.async {
-                NotificationCenter.default.post(name: Event.chatModified, object: nil, userInfo: [
-                    "chat_id": Int(data1),
-                ])
-            }
+
+            NotificationCenter.default.post(name: Event.chatModified, object: nil, userInfo: [
+                "chat_id": Int(data1),
+            ])
         case DC_EVENT_CHAT_EPHEMERAL_TIMER_MODIFIED:
             if accountId != dcAccounts.getSelected().id {
                 return
             }
             logger.info("📡[\(accountId)] ephemeral timer modified: \(data1)")
-            DispatchQueue.main.async {
-                NotificationCenter.default.post(name: Event.ephemeralTimerModified, object: nil, userInfo: [
-                    "chat_id": Int(data1),
-                ])
-            }
+
+            NotificationCenter.default.post(name: Event.ephemeralTimerModified, object: nil, userInfo: [
+                "chat_id": Int(data1),
+            ])
 
         case DC_EVENT_INCOMING_MSG:
-            DispatchQueue.main.async {
-                NotificationCenter.default.post(name: Event.incomingMessageOnAnyAccount, object: nil)
-            }
+            
+            NotificationCenter.default.post(name: Event.incomingMessageOnAnyAccount, object: nil)
             if accountId != dcAccounts.getSelected().id {
                 return
             }
             logger.info("📡[\(accountId)] incoming message \(data2)")
-            DispatchQueue.main.async {
-                NotificationCenter.default.post(name: Event.incomingMessage, object: nil, userInfo: [
-                    "message_id": Int(data2),
-                    "chat_id": Int(data1),
-                ])
-            }
+
+            NotificationCenter.default.post(name: Event.incomingMessage, object: nil, userInfo: [
+                "message_id": Int(data2),
+                "chat_id": Int(data1),
+            ])
 
         case DC_EVENT_CONTACTS_CHANGED:
             if accountId != dcAccounts.getSelected().id {
                 return
             }
             logger.info("📡[\(accountId)] contact changed: \(data1)")
-            DispatchQueue.main.async {
-                NotificationCenter.default.post(name: Event.contactsChanged, object: nil, userInfo: [
-                    "contact_id": Int(data1)
-                ])
-            }
+                
+            NotificationCenter.default.post(name: Event.contactsChanged, object: nil, userInfo: [
+                "contact_id": Int(data1)
+            ])
 
         case DC_EVENT_CONNECTIVITY_CHANGED:
             if accountId != dcAccounts.getSelected().id {
                 return
             }
             logger.info("📡[\(accountId)] connectivity changed")
-            DispatchQueue.main.async {
                 NotificationCenter.default.post(name: Event.connectivityChanged, object: nil)
```

Comment:
```suggestion
            NotificationCenter.default.post(name: Event.connectivityChanged, object: nil)
```

(just normalising whitespace)